components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -318,23 +362,39 @@ export default class AdvancedSettingsDisplay extends React.Component {
                                 type='radio'
                                 name='sendOnCtrlEnter'
                                 checked={ctrlSendActive[0]}
-                                onChange={this.updateSetting.bind(this, 'send_on_ctrl_enter', 'true')}
+                                onChange={this.handleCtrlEnterSetting.bind(this, 0)}
```

Comment:
Instead of binding a value to `this.handleCtrlEnterSetting`, it might be clearer to just use an arrow function containing the actual `this.updateSetting` calls here. That way, you wouldn't have to look up what 0, 1, and 2 each mean. It would look something like this for each of the settings.
```javascript
onChange={() => {
    this.updateSetting('send_on_ctrl_enter', 'true');
    this.updateSetting('code_block_ctrl_enter', 'true');
}}
```
We usually recommend against using arrow functions in the render method because it creates a new function on each render which can cause some small performance issues, but that's basically what's happening here with the bind.